BUILD.gn

```diff
@@ -121,6 +122,7 @@ repack("packed_extra_resources") {
     ":atom_resources",
     ":brave_resources",
     "app:brave_strings",
+    "//chrome/browser/resources:component_extension_resources_grit",
```

Comment:
shouldn't think be `//chrome/browser/resources:component_extension_resources`? I don't see a target for `component_extension_resources_grit`